doc/user/bgp.rst

```diff
@@ -250,7 +250,7 @@ internal or external.
    interface and address information. In that case default router ID value is
    selected as the largest IP Address of the interfaces. When `router zebra` is
    not enabled *bgpd* can't get interface information so `router-id` is set to
-   0.0.0.0. So please set router-id by hand.
+   0.0.0.0. So please set router-id by hand. Valid router IDs are between 0.0.0.0 and 223.255.255.255.
```

Comment:
If we are going with this https://github.com/FRRouting/frr/pull/6929, then the range is a bit bigger?

Reply:
Wrap to 80 columns please, otherwise lgtm

Reply:
The BGP Identifier is a 4-octet, unsigned, non-zero integer. According to this, we don't care much about the numbers except a sanity validation which is done underlay. No?